sample-apps/segmentation_spleen_tta/README.md

```diff
@@ -0,0 +1,14 @@
+# Segmentation - Spleen
```

Comment:
what is blocking to use the existing segmentation_spleen app for adding tta scoring..? why new APP...?

how about adding this into generic_segmentation app directly?

Reply:
Thanks for all your comments! I agree we should include all this in the generic App. My plan before Friday is to have a single deepedit generic App once we complete the updates on the deepedit transforms. For now, let's have this as a separate App.